topics/metabolomics/tutorials/qcxms-predictions/tutorial.md

```diff
@@ -0,0 +1,594 @@
+---
+layout: tutorial_hands_on
+title: Predicting EI+ mass spectra with QCxMS
+zenodo_link: 'https://zenodo.org/record/13327051'
+level: Introductory
+
+questions:
+- Can I predict QC-based mass spectra starting from SMILES?
+- How can I run those computationally heavy predictions? 
+- Can I take into account different conformers?
+
+objectives: 
+- To prepare SDF files for downstream analysis, starting from SMILES. 
+- To generate conformers and optimise them using semi-empirical methods.
+- To produce simulated mass spectra for a given molecule in MSP format.  
+
```

Comment:
Questions and objectives contain several abbreviation and highly technical terms. It would be nice to write these entry points into the tutorial in an easier to understand way